tensorflow/lite/kernels/kernel_util.cc

```diff
@@ -338,8 +338,8 @@ inline TfLiteStatus Quantize(TfLiteContext* context, float scale,
                              int32_t zero_point, float f, int32_t& q) {
   const float tmp = TfLiteRound(f / scale);
```

Comment:
Since tmp is already being cast to int32_t below, shouldn't we just do

const int32_t tmp = static_cast<int32_t>(TfLiteRound(f / scale));

I think that's safer than casting INT_MAX to float, as I don't think that is a lossless cast.

Reply:
Ignore this, I just noticed we're doing this specifically to check overflow :)